src/components/Score/index.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import {
+  Typography,
+  Paper,
+  Divider,
+  Button,
+  Box,
+  CircularProgress,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+// eslint-disable-next-line no-unused-vars
+import { BrowserRouter as Router, NavLink } from 'react-router-dom';
+
+const useStyles = makeStyles((theme) => ({
+  circularProgress: {
+    margin: '10px auto',
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  paper: {
+    maxWidth: '930px',
+    padding: '10px',
+    margin: '0 auto',
+  },
+  quizNavigation: {
+    width: '100%',
+    margin: '10px 0 0 auto',
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+const Score = ({ testTitle, score, numberOfTests }) => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Paper elevation={2} className={classes.paper}>
+        <Typography variant="h5" gutterBottom>
+          {testTitle} <Divider />
+        </Typography>
+        <Typography variant="h4" align="center">
+          {'Your result'}
+        </Typography>
+        <div className={classes.circularProgress}>
+          <CircularProgressWithLabel
+            score={score}
+            numberOfTests={numberOfTests}
+          />
+        </div>
+        <Box className={classes.quizNavigation} spacing={1}>
+          <Button color="primary" variant="contained">
+            <NavLink to={`../quiz/`} className="testNumberLink">
```

Comment:
why do you need to use JS expression as a parameter "to" ?

Reply:
Fixed. Update in next commit